KsApi/models/ProjectPLOTDataEnvelope.swift

```diff
@@ -0,0 +1,37 @@
+import Foundation
+import ReactiveSwift
+
+public struct ProjectPLOTDataEnvelope {
```

Comment:
nit: same suggestion here

```suggestion
public struct ProjectPledgeOverTimeDataEnvelope {
```